homeassistant/components/template/switch.py

```diff
@@ -59,38 +85,76 @@
 )
 
 PLATFORM_SCHEMA = SWITCH_PLATFORM_SCHEMA.extend(
-    {vol.Required(CONF_SWITCHES): cv.schema_with_slug_keys(SWITCH_SCHEMA)}
+    {vol.Required(CONF_SWITCHES): cv.schema_with_slug_keys(LEGACY_SWITCH_SCHEMA)}
 )
 
 SWITCH_CONFIG_SCHEMA = vol.Schema(
```

Comment:
In a separate PR, I'd suggest to rename the config entry schemas in template platforms to `xxx_CONFIG_ENTRY_SCHEMA` to make the code more readable